libusb/os/darwin_usb.c

```diff
@@ -1766,6 +1766,23 @@ static int submit_bulk_transfer(struct usbi_transfer *itransfer) {
     return darwin_to_libusb (ret);
   }
 
+  ret = (*(cInterface->interface))->GetAlternateSetting (cInterface->interface, &alternateSetting);
+
+  if (ret) {
```

Comment:
Might be nice if these 3 error messages could be unique, so that if/when anyone sees one, they know which it's from.